src/components/customer-io/email-entry-form.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import {track} from 'utils/analytics'
+import EmailForm from '../cta/email/email-form'
+import useCio from 'hooks/use-cio'
+import {useViewer} from 'context/viewer-context'
+import {requestSignInEmail} from 'utils/request-signin-email'
+
+const EmailEntryForm: React.FC<any> = ({
+  campaignAttribute,
+  siteLocation,
+  title,
+  subTitle,
+  buttonText = 'submit',
+}) => {
+  const {subscriber, cioIdentify} = useCio()
+  const {viewer} = useViewer()
+
+  const onSubmit = async (values: any) => {
+    const {email} = values
+
+    let id = subscriber?.id || viewer?.contact_id
+
+    if (!id) {
+      const {contact_id} = await requestSignInEmail(email)
```

Comment:
Customer.io isn't email-based, we use a "contact id" so if we don't know the user we need to make one based on the email.

For this specific PR it doesn't matter since we know the customer if they got here through our CTA, but if they stumble on it it will make them an "account"